app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -61,6 +63,10 @@ public GoogleSignInAccount getSavedSignedInAccount() {
         return savedSignedInAccount;
     }
 
+    public LeaderboardsClient getLeaderboardsClient() {
+        return leaderboardsClient;
+    }
+
```

Comment:
can we move this below the onCreate method?

Reply:
This one is a public method. I thought that we create our methods in this order:
1. Public
2. Protected
3. Private
Should it be like this?

Reply:
Yes they should. But in this case onCreate seems to be more important than the getter. You can leave it here if you want but you would need to have all the client getters there as well.

Reply:
Fixed.